test/hw7/test_hw7_1_find_occurrences_in_tree.py

```diff
@@ -1,67 +1,79 @@
 """TEST Homework 7.1."""
-from typing import Any
+from typing import Any, Dict
 
 import pytest
 
 from hw.hw7.hw7_1_tree import find_occurrences
 
 
+@pytest.fixture()
+def example_tree1() -> Dict:
+    return {
+        "first": ["RED", "BLUE"],
+        "second": {
+            "simple_key": ["simple", "list", "of", "RED", "valued"],
+        },
+        "third": {
+            "abc": "BLUE",
+            "jhl": "RED",
+            "complex_key": {
+                "key1": "value1",
+                "key2": "RED",
+                "key3": ["a", "lot", "of", "values", {"nested_key": "RED"}],
+            },
+        },
+        "fourth": "RED",
```

Comment:
Good fixtures, now let's place them to resource files (it is a bad practice to place JSON analogs to .py files)